charts/trento-server/charts/wanda/templates/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "wanda.fullname" . }}
+  labels:
+    {{- include "wanda.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "wanda.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "wanda.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "wanda.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: init
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "wanda.fullname" . }}-configmap
+            - secretRef:
+                name: {{ include "wanda.fullname" . }}-secret
+          args: ['eval', 'Wanda.Release.init()']
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "wanda.fullname" . }}-configmap
+            - secretRef:
+                name: {{ include "wanda.fullname" . }}-secret
+          ports:
+            - name: http
+              containerPort: {{ template "wanda.port" . }}
+              protocol: TCP
+          args: ['start']
+          # livenessProbe:
```

Comment:
I guess we can remove this comments, as wanda doesn't have any liveness probe

Reply:
I left it there as a reminder: shouldn't we actually add something like a liveness probe both to the server and wanda?